regression-tests/25-compile-c64/Makefile

```diff
@@ -0,0 +1,14 @@
+EXAMPLESDIR=../../examples
+TOOLSDIR=../../tools
+
+# build c64 tests. Many examples won't fit in memory or won't compile due to the
+# lack of int64_t in cc65, so only a subset of IPv4 is tested.
+EXAMPLES = irc/c64 \
+telnet-server/c64 \
+webbrowser/c64 \
+webserver/c64 \
+wget/c64
```

Comment:
Ah ok sorry. @tiagoshibata what we need is not this new file, but rather a new line in `16-compile-6502-ports/Makefile` to compile ipv6 on c64